include/revng/Support/EnumSwitch.h

```diff
@@ -0,0 +1,24 @@
+#pragma once
+
+//
+// This file is distributed under the MIT License. See LICENSE.md for details.
+//
+
+#include "cstddef"
```

Comment:
```suggestion
#include <cstddef>
```

Reply:
[Done](https://github.com/revng/revng/pull/98/commits/b82caf625950fd6f9a3d383d186f18ac4ae23dc9), sorry about this one.